googetdb/googetdb.go

```diff
@@ -36,21 +36,21 @@ const (
 )
 
 type gooDB struct {
-	db *sql.DB
+	DB *sql.DB
```

Comment:
Not blocking, but if the only reason you've exported DB here is so that you can call Close, it would be better to define

func (g *gooDB) Close() error { return g.db.Close() }

and use this instead.